meta-mender-core/recipes-mender/mender/mender_git.bb

```diff
@@ -59,7 +59,7 @@ def mender_license(branch):
         }
     else:
         return {
-                   "md5": "debbe5e440f2e65465e86b25fc7c9fcc",
+                   "md5": "0f993507ab8e2f5afa812bf033a4a24b",
```

Comment:
You say mender-artifact, but you changed mender! :smile:

Reply:
Well, yeh. Will update it :see_no_evil: . It is mender-artifact vendoring that has changed in mender. However, it is still not correct it seems.